src/linker/configuration.py

```diff
@@ -156,8 +187,43 @@ def _load_input_data_paths(input_data_specification_path: Path) -> List[Path]:
         return file_list
 
     @staticmethod
-    def _get_spark_requests(environment: Dict[str, Union[Dict, str]]) -> Dict[str, Any]:
-        spark = environment.get("spark", {})
-        if spark and not "keep_alive" in spark:
-            spark["keep_alive"] = False
-        return spark
+    def _get_computing_environment(environment: Dict[Any, Any]) -> str:
+        if not "computing_environment" in environment:
+            value = DEFAULT_ENVIRONMENT["computing_environment"]
+            logger.info(f"Assigning default value for computing_environment: '{value}'")
+        else:
+            value = environment["computing_environment"]
+        return value
+
+    @staticmethod
+    def _get_container_engine(environment: Dict[Any, Any]) -> str:
```

Comment:
Aren't these two methods identical to the else block of `_get_requests`?

Reply:
Not quite since these two are `str: str` vs `str: dict`. I'll clena these two up into one function at least but leave it as is unless yo uthink it's wroth putting more effort into this